framework/tests/flowexpression/ValueLiterals.java

```diff
@@ -2,4 +2,10 @@
 
 public class ValueLiterals {
     void test(@FlowExp("0") Object a, @FlowExp("0L") Object b) {}
+
+    void test2(@FlowExp("1000") Object a, @FlowExp("100L") Object b) {}
+    //:: error: (expression.unparsable.type.invalid)
+    void test3(@FlowExp("01000") Object a) {}
+    //:: error: (expression.unparsable.type.invalid)
+    void test4(@FlowExp("0100L") Object b) {}
```

Comment:
This is legal -- it represents the value 64.  What is the reason to forbid it?

Also, `-0L` is legal but is forbidden by the current regex.